packages/core/src/components/icon/icon.scss

```diff
@@ -1,10 +1,14 @@
 :host {
   @import "~@dso-toolkit/styling/scaffolding/normalize";
 
-  @import "~@dso-toolkit/styling/components/icon";
+  @import "~@dso-toolkit/styling/variables/icon";
 
   display: inline-block;
   height: $dso-icon-size;
   vertical-align: $dso-icon-vertical-align;
   width: $dso-icon-size;
 }
+
+.icon-container {
```

Comment:
Zie mijn losse comments, is deze misschien toch ook nodig in de klassieke HTML/CSS variant (nu ontploffen er sommige iconen). Maar: eigenlijk willen we deze ueberhaupt niet nodig hebben, toch?

Reply:
Deze container is alleen voor het web component nodig, we ontkomen er niet aan omdat we de het icon svg als string hebben.

Voor het CSS component verandert er niets